docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,266 @@
+The filter modular component allows you to increase the precision of your model's [data capture](/services/data/#data-capture) and determine which readings to store.
+This can help you to avoid sifting through unwanted data captures and ensures that only the data you're interested in is in Viam's cloud.
+
+## Hardware Requirements
+
+For this tutorial you'll need the following hardware components:
+
+- A computer
+- A webcam or external camera
+- A blue/green collar for increased precision _(optional)_
+
+## Set up
+
+After you [create and connect](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot) to your robot, here's how to get started:
+
+- Install the [Go binary](https://go.dev/dl/) on your local development computer.
+
+- Update [`viam-server`](/installation/manage/#update-viam-server).
+If you don't already have `viam-server` installed, follow [these directions](/installation/#install-viam-server) to install the most recent, stable version.
+
+- Clone the [Viam modular filter](https://github.com/viam-labs/modular-filter-examples) examples onto your robot's computer:
+
+   ```{class="command-line" data-prompt="$"}
+   git clone https://github.com/viam-labs/modular-filter-examples.git
+   ```
+
+### Configure the source code
+
+This tutorial makes use of [Viam's color filter example](https://github.com/viam-labs/modular-filter-examples/tree/main/colorfilter). However, you can modify the filter's source code or write your own filter for different components using [this guide](/extend/modular-resources/create/).
+
+Navigate to the `modular-filter-examples/colorfilter/module` directory in your terminal, and run the following command:
+
+```{class="command-line" data-prompt="$"}
+go build
+```
+
+This command will compile the source code in the colorfilter/module directory and generate an executable with the same name as the module, which is 'colorfilter'.
+
+{{< alert title="Tip" color="tip" >}}
+Go to the `colorfilter/module` directory of the color filter module you cloned and get the absolute path to your `colorfilter` module for later use by running:
+
+```{class="command-line" data-prompt="$"}
+realpath colorfilter 
+```
+
+{{< /alert >}}
+
+Read [this guide](https://docs.viam.com/extend/modular-resources/configure/#configure-a-local-module) for more information on configuring a local module.
+
+### Set up camera
+
+Navigate to your robot's page on the app and click on the **Config** tab.
+
+Add your robot's camera as a component by clicking **Create component** in lower-left corner of the page and typing in 'webcam' or specific the model you're using.
+
+![A photo of the webcam component named 'cam'](/tutorials/pet-photographer/webcam-component.png)
+
+For more information about the Camera component, you can refer to [this page](/components/camera/).
```

Comment:
See how above in my suggestion I pull in the link into the text as is? That has two advantages:
- You can remove a sentence and keep the content shorter
- The text is more descriptive of the link. Generally link text that says `link`, `here`, `this page` is not very useful to the reader for figuring out what will be behind the link, and then they need to read the surrounding text to get a clue. It's better to have the link text itself be descriptive
```suggestion
```